src/packageurl/contrib/url2purl.py

```diff
@@ -276,3 +276,23 @@ def build_sourceforge_purl(uri):
 @purl_router.route(cargo_pattern)
 def build_cargo_purl(uri):
     return purl_from_pattern('cargo', cargo_pattern, uri)
+
+
+@purl_router.route('https?://github.com.*/.*')
+def build_github_purl(uri):
+    path = unquote_plus(urlparse(uri).path)
```

Comment:
What is `unquote_plus` do?

Reply:
In above npm packages this have been used, so I think it will be standard practice to unquote the URLs before parsing them

Reply:
Yes ok this is fine.

Reply:
Do you have a test for that?

Reply:
Test case for which function ?

Reply:
A test for both quoted and unquoted URLs